config/aic.php

```diff
@@ -6,6 +6,8 @@
     'iiif_s3_endpoint' => env('IIIF_S3_ENDPOINT'),
     'sales_site_url' => env('SALES_SITE_URL', 'https://sales.artic.edu'),
     'vtour_bucket' => env('AWS_VTOUR_BUCKET'),
+    'http_username' => env('HTTPAUTH_USER'),
+    'http_password' => env('HTTPAUTH_PASS'),
```

Comment:
Could we make it so the config key and the env var look similar, eg. `'http_username' => env('HTTP_USERNAME'),`?

Reply:
I agree it would be clearer if the names were more similar. Heads up, we'll have to change the `.env`s in our test and prod environments to match the new name.